src/warplib/gatherWarplibImports.ts

```diff
@@ -8,10 +8,12 @@ export const warplibImportInfo = glob
   .reduce((warplibMap, pathToFile) => {
     const rawCairoCode = fs.readFileSync(pathToFile, { encoding: 'utf8' });
 
-    const importPath = pathToFile
+    let importPath = pathToFile
       .split('/')
-      .join('.')
-      .slice(0, pathToFile.length - '.cairo'.length);
+      // We need to remove the `src` folder where scarb store the contracts
+      .filter((dir) => dir !== 'src')
+      .join('.');
+    importPath = importPath.slice(0, importPath.length - '.cairo'.length);
```

Comment:
Regex seems like a better option. Something like `^warplib/src/\w+.cairo$` to match everything in between, then just use `.replace(path.sep, '.')`

Reply:
I did not fully understand what you meant here, specially how this regex would remove the `src` from the path. I've put a solution using regex nonetheless

Reply:
I meant regex with match method. You could extract everything between `src/` and `.cairo`, then do `warplib.${variable.replace(path.sep, '.')}`

Reply:
I see now. I am not sure I see it as a better solution than the current one. Using a match would also require to handle a possible `null` case.

Reply:
I see, then let's use slice: `importPath.slice('warplib/src/'.length,  - '.cairo'.length)`